src/components/dataTable/icon.scss

```diff
@@ -0,0 +1,37 @@
+table.md-table thead th {
```

Comment:
This approach, of having `icon.scss` inside table, feels kind of weird for me.

can't it be with `dataTable.scss`?